tests/core/pyspec/eth2spec/test/phase0/sanity/test_blocks.py

```diff
@@ -744,6 +744,27 @@ def test_deposit_in_block(spec, state):
     assert state.validators[validator_index].pubkey == pubkeys[validator_index]
 
 
+@with_all_phases
+@spec_state_test
+def test_invalid_double_same_deposit_same_block(spec, state):
```

Comment:
duplicate means "double same" essentially
```suggestion
def test_invalid_duplicate_deposit_same_block(spec, state):
```